src/pages/index.tsx

```diff
@@ -17,7 +17,7 @@ const Overview: NextPage = () => {
 
 	useEffect(() => {
 		getMarket();
-	}, [getMarket]);
+	}, []);
```

Comment:
add a comment explaining why we ignore the eslint warning here and why getmarket should not be added as a dependency again